contracts/UsingWitnet.sol

```diff
@@ -37,45 +37,73 @@ abstract contract UsingWitnet {
     /// @return A boolean telling if the request has been already resolved or not. Returns `false` also, if the result was deleted.
     function _witnetCheckResultAvailability(uint256 _id)
         internal view
+        virtual
         returns (bool)
     {
         return witnet.getQueryStatus(_id) == Witnet.QueryStatus.Reported;
     }
 
     /// Estimate the reward amount.
     /// @param _gasPrice The gas price for which we want to retrieve the estimation.
-    /// @return The reward to be included for the given gas price.
+    /// @return The reward to be included when either posting a new request, or upgrading the reward of a previously posted one.
     function _witnetEstimateReward(uint256 _gasPrice)
         internal view
+        virtual
         returns (uint256)
     {
         return witnet.estimateReward(_gasPrice);
     }
 
+    /// Estimates the reward amount, considering current transaction gas price.
+    /// @return The reward to be included when either posting a new request, or upgrading the reward of a previously posted one.
+    function _witnetEstimateReward()
+        internal view
+        virtual
+        returns (uint256)
+    {
+        return witnet.estimateReward(tx.gasprice);
+    }
+
     /// Send a new request to the Witnet network with transaction value as a reward.
     /// @param _request An instance of `IWitnetRequest` contract.
-    /// @return Sequential identifier for the request included in the WitnetRequestBoard.
+    /// @return _id Sequential identifier for the request included in the WitnetRequestBoard.
+    /// @return _actualReward Actual reward amount escrowed by the WRB until a result gets reported.
```

Comment:
```suggestion
    /// @return _reward Current reward amount escrowed by the WRB until a result gets reported.
```

Reply:
With "actual" I refer to the "real" reward sent to the WRB, which would be less than the `msg.value` sent by the caller in most cases. However, I agree that referring just to "reward" could be easier to follow.